### PR TITLE
feature: fabric enable tracing filters

### PR DESCRIPTION
**Pull request by szuecs**
feature: fabric enable tracing filters

### CODE REVIEW

File: cluster/manifests/fabric-gateway/deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{ $version := "master-200" }}
+{{ $version := "master-208" }}
```
Comment (RomanZavodskikh) [inline cluster/manifests/fabric-gateway/deployment.yaml]: I think both 208 and 209 are ok, but would like it to be 209.

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/manifests/fabric-gateway/deployment.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/manifests/fabric-gateway/deployment.yaml]: yes because of e2e weirdness I agree

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**